HW FuncApprox @FrancescoTucci and @gustavek

gustavekenedi opened the pull request.

Commit:
HW FuncApprox @FrancescoTucci and @gustavek #4

floswald commented on src/funcapp.jl:
```diff
@@ -1,74 +1,304 @@
+using ApproxFun
+using ApproXD
+using Plots
 
```
you need to put those things **inside** the module. you clearly didn't try to `include("run.jl")`. because that doesn't work!

gustavekenedi replied:
Apologies for the mistake. I've made a commit with the change and also commented savefig() for the first 2 functions since this resulted in an error message when I ran `include("run.jl")`.

Commit:
Update

Include loaded packages in module and comment on save fig for questions
1 and 2
